.github/workflows/screener-build.yml

```diff
@@ -0,0 +1,172 @@
+          EOT
+        env:
+          DEPLOYHOST: 'fluentuipr.z22.web.core.windows.net'
+          PR_NUMBER: ${{ github.event.number }}
+        if: ${{startsWith(github.ref, 'refs/pull/')}}
+
+      - name: Set env variables if there is not a PR
+        run: |
+          cat <<EOT >> artifacts/environment
+          ISPR=false
+          DEPLOYBASEPATH=heads/$SYSTEM_PULLREQUEST_TARGETBRANCH
+          DEPLOYURL=https://$DEPLOYHOST/heads/$SYSTEM_PULLREQUEST_TARGETBRANCH
+          BUILD_SOURCEBRANCHNAME=${{ github.ref_name }}
+          EOT
+        if: ${{!startsWith(github.ref, 'refs/pull/')}}
+
+      - name: Upload environment variables artifact
+        uses: actions/upload-artifact@v3
+        with:
+          name: env-artifact
+          path: artifacts/environment
+
+  ScreenerNorthStar:
+    runs-on: 'ubuntu-latest'
+
+    name: Screener @fluentui/react-northstar
+
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 14.18.1
+
+      - name: Log environment variables (Linux)
+        if: runner.os == 'Linux'
+        run: |
+          printenv | sort ;\
+          echo "SHELLOPTS $SHELLOPTS" ;\
+
+      - name: yarn
+        run: ./yarn-ci.sh
+        shell: bash
+
+      - name: build FUI N* VR Test
+        run: yarn workspace @fluentui/docs vr:build
+        env:
+          SCREENER_BUILD: 1
+
+      - uses: actions/upload-artifact@v3
+        with:
+          name: northstar-artifact
+          path: packages/fluentui/docs/dist
+
+  Screener:
+    runs-on: 'ubuntu-latest'
+
+    name: Screener @fluentui/react
+
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 14.18.1
+
+      - name: Log environment variables (Linux)
+        if: runner.os == 'Linux'
+        run: |
+          printenv | sort ;\
+          echo "SHELLOPTS $SHELLOPTS" ;\
+
+      - name: yarn
+        run: ./yarn-ci.sh
+        shell: bash
+
+      - name: build vr-tests storybook
+        run: yarn workspace @fluentui/vr-tests screener:build
+
+      - uses: actions/upload-artifact@v3
+        with:
+          name: screener-artifact
+          path: apps/vr-tests/dist/storybook
+
+  ScreenerVNext:
+    runs-on: 'ubuntu-latest'
+
+    name: Screener @fluentui/react-components
+
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 14.18.1
+
+      - name: Log environment variables (Linux)
+        if: runner.os == 'Linux'
+        run: |
+          printenv | sort ;\
+          echo "SHELLOPTS $SHELLOPTS" ;\
+
+      - name: yarn
+        run: ./yarn-ci.sh
+        shell: bash
+
+      - name: build vr-tests-react-components storybook
+        run: yarn workspace @fluentui/vr-tests-react-components screener:build
```

Comment:
**suggestion:** can we check if these builds need to be executet at all ( if there are no related changes) ? Im aware we do this for actual VR runs. can be implemented as PR followup. cc @ling1726

Reply:
That's actually an existing problem even in our current ADO pipeline and we discussed that it should be fixed. @ralucapelin can you please link the issue for this? (that the build happens even if the screener run will be skipped)

Reply:
Yes, it's issue #23905

Reply:
awesome, resolving this thread. ty!